azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/viewmodel/ChatScreenViewModel.kt

```diff
@@ -63,3 +60,29 @@ internal fun buildChatScreenViewModel(
         navigationStatus = store.getCurrentState().navigationState.navigationStatus,
     )
 }
+
+private fun getUnReadMessagesCount(
+    store: AppStore<ReduxState>,
+    messages: List<MessageInfoModel>,
+): Int {
+    var unreadMessagesCount = 0
+
+    val lastReadId = store.getCurrentState().chatState.lastReadMessageId
+    val lastSendId = store.getCurrentState().chatState.lastSendMessageId
+    var itr = 0
+    while (itr < messages.size) {
+        if (lastReadId.isEmpty()) {
+            break
+        }
```

Comment:
when lastReadId is empty, unreadMessageCount should be the full message size. 
Right now it will return 0.